blog.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <img src="images/adoptadog-logo.svg" alt="logo">
+        <div>Seattle's premiere adoption agency</div>
+      </div>
+      <div class="cart">
+        <div class="total-cart">
+          <i class="fas fa-shopping-cart"></i>
+          <p>$1234.56</p>
+        </div>
+        <button type="button" name="button" id="cart-button"><a href="checkout.html">Checkout</a></button>
+      </div>
+    </header>
+    <nav class="main-nav">
+      <a href="index.html">Home</a> /
+      <a href="browse.html">Dogs</a> /
+      <a href="blog.html">Blog</a>
+    </nav>
+    <div class="container">
+
+      <div class="canvas">
```

Comment:
You might not need this wrapping div, the div with the container might be enough